yt_dlp/extractor/fujitv.py

```diff
@@ -13,61 +17,245 @@ class FujiTVFODPlus7IE(InfoExtractor):
+                    'person_name': '渡辺　菜生子'
+                },
+                {
+                    'person_id': 'M04-0312',
+                    'person_name': '菊池　正美'
+                },
+                {
+                    'person_id': 'M93-2251',
+                    'person_name': '飛田　展男'
+                },
+                {
+                    'person_id': 'SMB-070004',
+                    'person_name': '田野　めぐみ'
+                },
+                {
+                    'person_id': 'SMB-070005',
+                    'person_name': 'カシワクラツトム'
+                },
+                {
+                    'person_id': 'W93-2156',
+                    'person_name': 'ならはし　みき'
+                },
+                {
+                    'person_id': 'M93-2118',
+                    'person_name': '茶風林'
+                },
+                {
+                    'person_id': 'W93-2004',
+                    'person_name': '中　友子'
+                },
+                {
+                    'person_id': 'SMB-135480',
+                    'person_name': '本井　えみ'
+                },
+                {
+                    'person_id': 'M98-0260',
+                    'person_name': '陶山　章央'
+                },
+                {
+                    'person_id': 'W96-0379',
+                    'person_name': '永澤　菜教'
+                },
+                {
+                    'person_id': 'M09-1345',
+                    'person_name': '沼田　祐介'
+                },
+                {
+                    'person_id': 'W00-0610',
+                    'person_name': '真山　亜子'
+                },
+                {
+                    'person_id': 'SMB-089907',
+                    'person_name': '木村　匡也'
+                }
+            ],
+            'share_url': 'https://fod.fujitv.co.jp/title/5d40/5d40810269/',
+            'sales_type': [
+                'plus7'
+            ],
+            'is_teaser': false,
+            'is_live': false,
+            'is_download': false,
+            'is_sold_together': false,
+            'is_vm': false,
+            'genre': {
+                'genre_id': 'AN',
+                'genre_name': 'アニメ',
+                'genre_eng_name': 'anime'
+            },
+            'minogashi_pairing_ep_id': '5d40110269',
+            'ep_release_date': '2025-10-05T18:30:00+09:00',
+            'ppv_status': 'none',
+            'priority_number': 2660,
+            'sort_number': 2660,
+            'purchase_end': '2025-10-12T18:00:00+09:00',
+            'ep_hash_id': '160a8cf865c71879f90623298c3eaffcavod',
+            'is_lite_course_lock': false
+        }
+        '''
+
+        detail = json_response
+        # UA must be set to Android TV to get the settings URL otherwise, the URL wont appear
+        # We dont know which UA is required to get the `android_tv` URL (if we manage to find out, we can use it here)
+        settings_json = self._download_json(
+            self._AUTH_API,
+            video_id,
```

Comment:
If a variable is only used a single time we might as well inline it
```suggestion
            'https://fod.fujitv.co.jp/apps/api/1/auth/contents/web',video_id,
```